### PR TITLE
Fixed bridge rendering glitches

### PR DESCRIPTION
**Pull request by Mailaender**
As suggested in https://github.com/OpenRA/OpenRA/issues/9322.

### CODE REVIEW

**Comment by abcdefg30** [comment]
I can still reproduce #9322.

**Comment by Mailaender** [comment]
Are you sure as I can't.

**Comment by abcdefg30** [comment]
bridgebug

**Comment by Mailaender** [comment]
Okay, do we still want this?

**Comment by abcdefg30** [comment]
Well, the changes still look reasonable to me. :+1:

File: OpenRA.Mods.Common/Traits/CustomSelectionSize.cs [hunk 1]
```diff
@@ -13,9 +13,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
-	[Desc("Special case trait for unselectable actors that need to define targetable area bounds",
-	"for special cases like C4, engineer repair and tooltips.",
-	"Examples: bridge huts and crates.")]
+	[Desc("Special case trait for actors that need to define targetable area and rendering bounds manually.")]
```
Comment (reaperrr) [inline OpenRA.Mods.Common/Traits/CustomSelectionSize.cs]: Actually, to my knowledge this does *only* affect targetable area, not rendering bounds.

File: OpenRA.Mods.Common/Traits/Render/AutoSelectionSize.cs [hunk 2]
```diff
@@ -13,6 +13,7 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Desc("Automatically calculates the targetable area and rendering boundaries from the sprite size.")]
```
Comment (reaperrr) [inline OpenRA.Mods.Common/Traits/Render/AutoSelectionSize.cs]: Same here.

File: OpenRA.Mods.Common/Traits/CustomSelectionSize.cs (hunk 1 above)
Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/CustomSelectionSize.cs]: That is what I thought as well until I was lectured by @pchote in https://github.com/OpenRA/OpenRA/pull/9203 so I wanted to share that knowledge.

Comment (pchote) [inline OpenRA.Mods.Common/Traits/CustomSelectionSize.cs]: It affects everything that queries the ScreenMap, which includes selecting the actors that should be rendered.

Comment (reaperrr) [inline OpenRA.Mods.Common/Traits/CustomSelectionSize.cs]: You learn something new every day... ok then.

Comment (pchote) [inline OpenRA.Mods.Common/Traits/CustomSelectionSize.cs]: Note that that *is* different to the rendering bounds.

Comment (Mailaender) [inline OpenRA.Mods.Common/Traits/CustomSelectionSize.cs]: How would you call it instead here then?

**Comment by Mailaender** [comment]
Let's not merge this yet. I am not yet sure about all effects this has or if it is needed at all.

**Comment by pchote** [comment]
This "doesn't work" because you haven't changed all the bridges.

**Comment by Mailaender** [comment]
Should be fixed now.

**Comment by reaperrr** [comment]
Can confirm this fixes the issue now :+1:

**Comment by RoosterDragon** [comment]
Much better :+1:

**Comment by RoosterDragon** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/5ed9f6b9f50d5e11d157eafd2646374ece2bde73...4f882ebb9d8369666a4bf68d2767aeb38c8d186c)

**Pull request merged**